docs/UserGuide.adoc

```diff
@@ -29,174 +29,452 @@ apps.
+* `find Betsy Tim John` +
+Returns any person having names `Betsy`, `Tim`, or `John`
+
 ==== Delete contact: `contacts delete`
 
+Deletes the specified person in Cow. +
 Format: `contacts delete INDEX`
 
+****
+* Deletes the person at the specified `INDEX`.
+* The index refers to the index number shown in the displayed person list.
+* The index *must be a positive integer* 1, 2, 3, ...
+****
+
+Examples:
+
+* `contacts list` +
+`contacts delete 2` +
+Deletes the 2nd person in Cow
+* `contacts find Jane` +
+`contacts delete 1` +
+Deletes the 1st person in the list of people resulting from the `contacts find`
+command
+
 // tag::assigntasktocontact[]
 ==== Assign tasks to contact: `contacts assign`
 
+Assigns a task to a specified person in Cow. +
 Format: `contacts assign c/CONTACT_INDEX k/TASK_INDEX`
 
+****
+* Assigns the person at the specified `CONTACT_INDEX` to the task at the specified
+`TASK_INDEX`.
+* `CONTACT_INDEX` refers to the index number shown in the displayed person list.
+* `TASK_INDEX` refers to the index number shown in the displayed task list.
+* These indices *must be a positive integer* 1, 2, 3, ...
+****
+
+Examples:
+
+* `contacts list` +
+`tasks list` +
+`contacts assign c/2 k/1` +
+Assigns the 2nd person in Cow to the 1st task in Cow
+* `contacts find Alex` +
+`tasks list` +
+`contacts assign c/1 k/3`
+Assigns the 1st person in the list of people resulting from the `contacts find`
+command to the 3rd task in Cow
+
 ==== Unassign tasks from contact: `contacts unassign`
 
+Unassigns a task from a specified person in Cow. +
 Format: `contacts unassign c/CONTACT_INDEX k/TASK_INDEX`
 // end::assigntasktocontact[]
 
+****
+* Unassigns an *already existing assignment* between the person at the specified
+`CONTACT_INDEX` to the task at the specified `TASK_INDEX`.
+* `CONTACT_INDEX` refers to the index number shown in the displayed person list.
+* `TASK_INDEX` refers to the index number shown in the displayed task list.
+* These indices *must be a positive integer* 1, 2, 3, ...
+****
+
+Examples:
+
+* `contacts list` +
+`tasks list` +
+`contacts unassign c/2 k/1` +
+Unassigns the 2nd person in Cow from the 1st task in Cow
+* `contacts find Alex` +
+`tasks list` +
+`contacts assign c/1 k/3`
+Unassigns the 1st person in the list of people resulting from the `contacts find`
+command from the 3rd task in Cow
+
 ==== Show tasks assigned to contact: `contacts assigned`
 
-Format: `contacts assigned CONTACT_ID`
+Displays all tasks assigned to the specified contact in the tasks pane.
+Format: `contacts assigned INDEX`
 
-This will display all tasks assigned to the contact in the tasks pane.
+****
+* Displays all tasks assigned to the person at the specified `INDEX`.
+* The index refers to the index number shown in the displayed person list.
+* The index *must be a positive integer* 1, 2, 3, ...
+****
+
+[TIP]
+The person specified at `INDEX` will be highlighted, and the tasks assigned are
+displayed on the displayed tasks list
```

Comment:
I think the tips need to end with periods. This issue probably exists throughout the UG - there are some with periods and some without.